pymeasure/instruments/attocube/anc300.py

```diff
@@ -255,3 +247,65 @@ def stop_all(self):
             attribute = getattr(self, attr)
             if isinstance(attribute, Axis):
                 attribute.stop()
+
+    def __init__(self, host, axisnames, passwd, query_delay=0.05, **kwargs):
```

Comment:
Please add a note to the CHANGES.rst file regarding the change, that the Attocube uses a resource string and not a hostname any more.